examples/openfoam_supersonic_wedge.py

```diff
@@ -41,20 +41,25 @@ def create_new_case(case_dir):
     #Creates the case
     return Case(case_dir)
 
-def write_initial_control_dict(case):
+def write_control_dict(case, n_iter):
     """Sets up the control dictionary.
     In this example we use the rhoCentralFoam compressible solver"""
 
+    if isinstance(n_iter, int) != True:
```

Comment:
```python
if foo == True:
   # ...
```
is better written as
```python
if foo:
    # ...
```

But I don't think this is the right thing to do since passing ``n_iter = 1.0`` should work. I'd just remove this entire check.